src/Gui/SoFCColorBar.cpp

```diff
@@ -78,6 +80,32 @@ void SoFCColorBarBase::GLRenderBelowPath(SoGLRenderAction *  action)
     SoSeparator::GLRenderBelowPath(action);
 }
 
+float SoFCColorBarBase::getBoundingWidth(const SbVec2s& size)
+{
+    // These are the same camera settings for front nodes as defined in the 3d view
+    SoOrthographicCamera* cam = new SoOrthographicCamera;
+    cam->position = SbVec3f(0, 0, 5);
```

Comment:
At first many thanks! Tested in a 15 in Hi-DPI laptop and a big screen (normal DPI) and it works fine.

However, I would like to get the numbers documented.  For example why is Z 5 here?

Reply:
It's the same camera configuration as here: https://github.com/FreeCAD/FreeCAD/blob/master/src/Gui/View3DInventorViewer.cpp#L386

The Z value of 5 is irrelevant and any other value > 0.0 probably works, too. The more important value is `cam->height = 10;` because this means that the normalized coordinates are in the range [-5, +5] x [-5*ratio, +5*ratio] if ratio > 1 and [-5*ratio, +5*ratio] x [-5, +5] if ratio < 1. This explains the numbers like 4.0, 4.5 or 5.0 in the code.

Reply:
Many thanks. Now I understand the code better. I documented this now: https://github.com/FreeCAD/FreeCAD/commit/537acfdddd